src/include/rbd/librbd.h

```diff
@@ -144,6 +144,8 @@ enum {
   RBD_IMAGE_OPTION_JOURNAL_ORDER = 5,
   RBD_IMAGE_OPTION_JOURNAL_SPLAY_WIDTH = 6,
   RBD_IMAGE_OPTION_JOURNAL_POOL = 7,
+  RBD_IMAGE_OPTION_FEATURES_SET = 8,
```

Comment:
... plus I think having RBD_IMAGE_OPTION_FEATURES and RBD_IMAGE_OPTION_FEATURES_SET is confusing since they essentially do the same thing. What if I specify both? What if I specify RBD_IMAGE_OPTION_FEATURES_SET and MASK? Too many unclear behaviors. Whereas with just the _MASK, it's clear it is always applied last to your feature set before the image is created.